python/mxnet/ndarray/sparse.py

```diff
@@ -179,19 +179,27 @@ def asnumpy(self):
         """
         return self.tostype('default').asnumpy()
 
-    def astype(self, dtype):
+    def astype(self, dtype, copy=True):
         """Returns a copy of the array after casting to a specified type.
         Parameters
         ----------
         dtype : numpy.dtype or str
             The type of the returned array.
+        copy : bool
+            Default `True`. By default, astype always returns a newly
+            allocated ndarray on the same context. If this is set to
+            False, and the dtype requirement is satisfied,
```

Comment:
It's unclear what the `dtype requirement` here is from the doc. Please fix.

Reply:
I agree. However I used the exact same wording as the numpy docs: https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.ndarray.astype.html
```
copy : bool, optional: By default, astype always returns a newly allocated array. If this is set to false, and the dtype, order, and subok requirements are satisfied, the input array is returned instead of a copy.
```
I suggest switching to:
``` 
If this is set to False, and the dtype requested is the same as the array's dtype,
```

Reply:
Looks good